tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1093,7 +1094,7 @@ def test_positive_end_to_end(self, fake_manifest_is_set):
             17. Create a new subnet
             18. Create a new domain
             19. Create a new hostgroup and associate previous entities to it
-            20. Provision a client
+            20. Provision a client  **  NOT CURRENTLY PROVISIONING
```

Comment:
Is string `** NOT CURRENTLY PROVISIONING` intentional ?

Reply:
yep, the test doesn't actually provision through Satellite.